schema/config/email.json

```diff
@@ -0,0 +1,46 @@
+{
+    "type": "object",
+    "properties": {
+        "engine": {
+            "type": "string",
+            "minLength": 1
+        },
+        "from": {
+            "type": "string",
+            "format": "email",
+            "minLength": 1
+        }
+    },
+    "required": [
+        "engine",
+        "from"
+    ],
+    "oneOf": [
+        {
+            "properties": {
+                "engine": {
+                    "enum": [
+                        "ses"
```

Comment:
Wondering if we want to put enums for engines, what if they want to use something else? tThen they need to modify the code just to add a different engine.